presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -294,9 +293,28 @@ protected void assertAccessDenied(@Language("SQL") String sql, @Language("RegExp
     }
 
     protected void assertAccessDenied(
+            @Language("SQL") String sql,
+            @Language("RegExp") String exceptionsMessageRegExp,
+            @Language("RegExp") String exceptionsCauseMessageRegExp,
+            TestingPrivilege... deniedPrivileges)
+    {
+        assertAccessDenied(getSession(), sql, ".*" + exceptionsMessageRegExp + ": Access Denied: " + exceptionsCauseMessageRegExp, ".*Access Denied: " + exceptionsCauseMessageRegExp, deniedPrivileges);
```

Comment:
Please do not use `.*` at the begin and end of error message regexp. We need be sure if there is nothing extra here.

See: 
```
           throw new PrestoException(PERMISSION_DENIED, "View owner does not have sufficient privileges: " + e.getMessage(), e);
```
There is a suffix added that you didn't verify. Please fix this.

Reply:
I shouldn't be that strict. Maybe `.*` at the beginning might be needed, but not at the end.

Reply:
It's better not to use `.*` at all. If there is a spercific part that's variable and cannot be matched (eg table name), using `\w+` is better.

Reply:
done.